src/Fixer/Alias/PowToExponentiationFixer.php

```diff
@@ -70,6 +72,21 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Converts \'pow()\' to \'**\'.',
+            null,
+            array(new CodeSample("<?php\n pow(\$a, 1);")),
+            null,
+            null,
+            'Risky when the function \'pow()\' function is overloaded.'
```

Comment:
overloaded function is when a function has multiple possible interfaces.
like `pow(int):int` and `pow(float):float`
it's not supported by PHP language syntax.

here, you meant that function is overridden, so one has replaced default implementation of it.